### PR TITLE
Kp 8837 portal update

### PR DESCRIPTION
**Pull request by mmatthiesencsc**
This was a maintenance update., updating the OS, Wordpress+plugins.
- IPs had changed and could not be reclaimed, so new IPs are assigned to prod/pre-prod vms.
- The mariadb repo url is not correctly pointing to major.minor.

### CODE REVIEW

File: inventories/development [hunk 1]
```diff
@@ -2,12 +2,12 @@ all:
...
       ansible_python_interpreter: python
     portal:
-      ansible_host: 195.148.21.204
+      ansible_host: 86.50.170.142
       ansible_user: almalinux
     portal-prod:
-      ansible_host: 128.214.255.125
-      ansible_user: cloud-user
+      ansible_host: 195.148.21.204
+      ansible_user: almalinux
```
Comment (Traubert) [inline inventories/development:13]: This change is unrelated in view of the PR comment. It's obviously consistent and correct, but can this break anything?

Comment (mmatthiesencsc) [reply inventories/development:13]: No, on the contrary. The old cloud-user refers to CENTOS7 machines. almalinux is the new account for AlmaLinux9. And the IP needed to change because the old one is in use at another VM which is in production (remsaa).

Comment (Traubert) [reply inventories/development:13]: Ah, right. Then all ok.

**Review by Traubert — APPROVED** [review]
